Dockerfile

```diff
@@ -1,50 +1,48 @@
-FROM golang:1.16-alpine as builder
+FROM restreamio/gstreamer:1.18.5.0-dev as builder
 
 WORKDIR /workspace
 
+RUN apt-get update && apt-get install -y golang
+
 # Copy the Go Modules manifests
-COPY service/go.mod go.mod
-COPY service/go.sum go.sum
+COPY go.mod .
+COPY go.sum .
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
 RUN go mod download
 
 # Copy the go source
-COPY service/cmd/ cmd/
-COPY service/pkg/ pkg/
-COPY service/version/ version/
-
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o livekit-recorder-service ./cmd/server
+COPY cmd/ cmd/
+COPY pkg/ pkg/
+COPY version/ version/
 
-FROM buildkite/puppeteer:latest
-
-COPY --from=builder /workspace/livekit-recorder-service /livekit-recorder-service
+WORKDIR /workspace
+RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o livekit-recorder ./cmd/server
 
-# Install pulse audio
-RUN apt-get -qq update && apt-get install -y pulseaudio
+FROM restreamio/gstreamer:1.18.5.0-dev
```

Comment:
does it makes sense to use this as the deploy image? how big is it?

you might be ok just apt-get gstreamer on a blank image.